CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ CHANGELOG
 3.x.x
 -----
 
+**ENHANCEMENTS**
+- Add new configuration parameter `DeletionPolicy` for EFs and FSx for Lustre shared storage 
```

Comment:
EFs --> EFS

Reply:
I am going to merge this PR first and address this change in my PR